components/meter/meter-circle.js

```diff
@@ -41,6 +50,14 @@ class MeterCircle extends MeterMixin(RtlMixin(LitElement)) {
 		:host([foreground-light]) .d2l-meter-circle-text {
 			fill: white;
 		}
+		.d2l-meter-circle-text-secondary {
+			color: var(--d2l-color-ferrite);
+			font-size: 0.6rem;
```

Comment:
Looks like `font-size: 0.6rem` should already be applied by the `d2l-body-small` on this element.